inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/fetcher/kafka/KafkaMultiTopicsFetcher.java

```diff
@@ -395,4 +395,16 @@ private void fetchFromKafka() throws Exception {
             }
         }
     }
+
+    /**
+     * negativeAck Offset
+     *
+     * @param msgOffset String
+     */
+    @Override
+    public void negativeAck(String msgOffset) throws Exception {
```

Comment:
Kafka does support negativeAck

Reply:
Kafka do not support negative ack.
Just pause topic consumer for 10 minutes.